lib/kino/input.ex

```diff
@@ -268,12 +333,19 @@ defmodule Kino.Input do
     * `:min` - the minimum datetime value (in UTC)
 
     * `:max` - the maximum datetime value (in UTC)
+
+    * `:debounce` - the option to modify the change event behavior.
+      Accepts an integer timeout (in milliseconds) or `"blur"`.
+      If an integer, the event emits after the specified delay.
+      If `"blur"`, it emits when the field is deselected by the user.
+      Defaults to `"blur"`
   """
   @spec utc_datetime(String.t(), keyword()) :: t()
   def utc_datetime(label, opts \\ []) when is_binary(label) and is_list(opts) do
```

Comment:
This should be `date`, but let's not do this for dates for now :)

Reply:
👍